packager/mpd/base/mpd_builder_unittest.cc

```diff
@@ -20,14 +22,19 @@ namespace shaka {
 
 namespace {
 
-class TestClock : public base::Clock {
+class TestClock : public Clock {
  public:
-  explicit TestClock(const base::Time& t) : time_(t) {}
-  ~TestClock() override {}
-  base::Time Now() override { return time_; }
+  explicit TestClock(std::tm& utc_time) {
+    std::tm local_time = utc_time;
+    std::time_t utc_time_t = std::mktime(&local_time);
+    std::time_t offset = utc_time_t - std::mktime(std::gmtime(&utc_time_t));
+    mock_time_ = std::chrono::system_clock::from_time_t(utc_time_t + offset);
```

Comment:
In muxer, I have the clock abstraction return uint64_t in seconds since 1970 UTC.  (Essentially the same as time_t.)  Depending on how it is consumed, you could do the same here, and avoid time_point and std::chrono.